torch/csrc/distributed/c10d/Utils.cpp

```diff
@@ -284,6 +292,9 @@ int connect(
         "host not found: " + std::string(gai_strerror(err)));
   }
 
+  VLOG(1) << "connect() to address " << address
```

Comment:
Why don't see use a more conventional logging mechanism here? I would assume this would be an INFO or DEBUG level log output.